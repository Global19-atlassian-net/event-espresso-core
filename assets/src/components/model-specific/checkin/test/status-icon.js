/**
 * External imports
 */
import { shallow } from 'enzyme';

/**
 * Internal imports
 */
import CheckInStatusIcon from '../status-icon';

describe( 'CheckInStatusIcon', () => {
	const rendered = ( props ) => shallow( <CheckInStatusIcon { ...props } /> );
	it( 'renders as expected for default props', () => {
		const wrapper = rendered();
		expect( wrapper ).toMatchSnapshot();
	} );
} );
