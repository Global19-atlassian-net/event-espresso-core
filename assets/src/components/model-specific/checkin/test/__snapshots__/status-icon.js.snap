// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckInStatusIcon renders as expected for default props 1`] = `
<Dashicon
  className="ee-red"
  icon="no-alt"
/>
`;
